frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -79,13 +81,30 @@ export const FormLabel = ({
 
 FormLabel.Label = ChakraFormLabel
 
-FormLabel.Description = ({ children, ...props }: TextProps): JSX.Element => {
+const FormLabelDescription = ({
+  children,
+  ...props
+}: TextProps): JSX.Element => {
+  const field = useFormControlContext()
```

Comment:
this hook is not documented, can we add a comment here to say what it does pls?

Reply:
Resolved in 2547093. Sorry for the delay.